server/package.json

```diff
@@ -6,15 +6,26 @@
   "scripts": {
     "start": "node dist/server.js",
     "watch": "concurrently \"npx tsc -w\" \"nodemon dist/server.js\"",
-    "build": "npx tsc"
+    "build": "npx tsc",
```

Comment:
Let's discuss how's this build generated

Reply:
I added the ts-node package in my latest commit, so now there's no need for this build step and /dist folder